src/main/java/io/scif/services/DefaultFormatService.java

```diff
@@ -193,7 +194,9 @@ public void addComponents(final Format format) {
 		checkerMap().put(format.getCheckerClass(), format);
 		parserMap().put(format.getParserClass(), format);
 		readerMap().put(format.getReaderClass(), format);
-		writerMap().put(format.getWriterClass(), format);
+		if (format.getWriterClass() != DefaultWriter.class) {
```

Comment:
Let's split this to its own commit since this is functionally fixing a bug, in my opinion, and separate from the message update.

The easiest way to do this will require using [git add -p](https://stackoverflow.com/questions/1085162/commit-only-part-of-a-files-changes-in-git)

Reply:
whoooops this is what happens when I do a review solely through "Files changed". This is my mistake